cabal-testsuite/PackageTests/Check/ConfiguredPackage/Sanity/NoDupNames/cabal.out

```diff
@@ -1,3 +1,4 @@
 # cabal check
 Warning: The package will not build sanely due to these errors:
 Warning: Duplicate sections: dup. The name of every library, executable, test suite, and benchmark section in the package must be unique.
+Warning: Hackage would reject this package.
```

Comment:
In the previous implementation the warning was truncated because of an uncaught `error`.